tests/acceptance/routing-test.js

```diff
@@ -0,0 +1,101 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from 'ember-twiddle/tests/helpers/start-app';
+
+let application;
+
+module('Acceptance | routing', {
+  beforeEach: function() {
+    application = startApp();
+  },
+
+  afterEach: function() {
+    Ember.run(application, 'destroy');
+  }
+});
+
+test('Able to do routing in a gist', function(assert) {
+  const login = "Gaurav0";
+  const gist_id = "35de43cb81fc35ddffb2";
+  const commit = "f354c6698b02fe3243656c8dc5aa0303cc7ae81c";
+
+  function create_file(filename, content) {
```

Comment:
The main reason I haven't done so is that this uses closure over login, gist_id, and commit.